plugins/3MFReader/ThreeMFWorkspaceReader.py

```diff
@@ -228,11 +234,14 @@ def preRead(self, file_name, show_dialog=True, *args, **kwargs):
         self._resolve_strategies = {k: None for k in resolve_strategy_keys}
         containers_found_dict = {k: False for k in resolve_strategy_keys}
 
+        # Check whether the file is a UCP, which changes some import options
+        is_ucp = file_name.endswith('.ucp')
```

Comment:
It's fine for now, but for the future i would just look for the extra settings file. That way it's already compatible if we ever decide to merge it.

Reply:
This would be cleaner indeed :+1: